template-parts/dynamic-header.php

```diff
@@ -56,7 +56,7 @@
 			</nav>
 			<div class="site-navigation-toggle-holder <?php echo esc_attr( hello_show_or_hide( 'hello_header_menu_display' ) ); ?>">
 				<div class="site-navigation-toggle" role="button" tabindex="0">
-					<i class="eicon-menu-bar" aria-hidden="true"></i>
+					<span class="site-navigation-toggle-icon"></span>
```

Comment:
@rami-elementor 

Why not keep the same `i` instead of `span` so we don't risk breaking a layout?

Reply:
This release already makes several markup changes.

Reply:
But if we can reduce the changes it is better, especially in a sensitive area like the menu